.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
what is this file and can we get rid of it?

Reply:
the file is removed in the latest commit